### PR TITLE
Zixi output plugin

### PR DESCRIPTION
**Pull request by zixi-roy**
Ive split the change into 4 commits, it should be more easier to see all the changes like that.

1. Insert zixi-output support
2. Allow "simple rtmp" to forward stream via zixi connections
3. Insert encoder feedback and video input decimation (to control target bitrate on the fly)
4. Add target bitrate changing while encoder is active in libx264

### CODE REVIEW

**Comment by jp9000** [comment]
Hi there.  Before I go over this, I would like you to first read this thoroughly from top to bottom, and then come back and go over your pull request:

https://obsproject.com/blog/how-to-contribute-to-open-source

**Comment by derrod** [comment]
Hey there, it appears you included binary files (two .libs and one .dylib) file in this PR. We'd rather avoid that. Also where can one find the source code used to create those binaries?

**Comment by jp9000** [comment]
I'm closing this for now to clean up the pull requests section -- I am not outright rejecting the proposal to add zixi support, so I don't recommend deleting your branch, but please get back to me (here, discord, email, or PM) when your company has had time to discuss licensing issues with your code.  I cannot merge code that includes pre-compiled static libraries.

All code contributions to this project must be in source code form, and must be compatible with our license.

So basically put this on hold until zixi has discussed and/or decided how to deal with this first.
